packages/website/src/components/magazine/hero-section-slice/HeroSectionSlice.js

```diff
@@ -7,11 +11,8 @@ import {
   HeroSectionBg,
   HeroSectionWrapper
 } from './HeroSectionSliceElements'
-import { GatsbyImage } from 'gatsby-plugin-image'
-import { Link } from 'gatsby'
-import AnimateIn from '../../animations/AnimateIn'
 
-const HeroSectionSlice = ({ date, mainImage, title, info, link }) => {
+const HeroSectionSlice = ({ date, mainImage, title, info, link, buttonName, buttonTarget = '_blank', showButton = false }) => {
```

Comment:
here , why we are using a `default parameters` if we know for sure that the arguments are neither ` missing ` nor ` undefined `

Reply:
Did the other way around.